charts/stable/mosquitto/values.yaml

```diff
@@ -9,7 +9,7 @@ image:
   # -- image repository
   repository: eclipse-mosquitto
   # -- image tag
-  tag: 2.0.11
+  tag: 2.0.14
```

Comment:
Please change this to the following so that in the future it will follow appVersion by default:

```yaml
  # -- image tag
  # @default -- chart.appVersion
  tag:
```

Reply:
Makes sense, done 👍